helm/generic-node/values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+name: "generic-node"
+
+testnetName: "testnet"
+
+image:
+  repository: minaprotocol/mina-daemon
+  pullPolicy: IfNotPresent
+  tag: 1.2.1alpha1-1f98b8b-devnet
+
+features:
+  logsnarkworkgossip:
+    enabled: true
+  enablepeerexchange:
+    enabled: true
+  runtimeConfig:
+    enabled: false
+  maxConnections: 200
+  logTxnPoolGossip:
+    enabled: false
+  exposeGraphql:
+    enabled: false
+  uploadBlocksToGCloud:   # Added this
+    enabled: true
+  customEntrypoint:
+    enabled: false
+    command: ""
+  prometheus:
+    enabled: true
+  persistence:
+    enabled: true
+    name: "generic-node-volume"
```

Comment:
Document that this needs a unique name per deploy, maybe automate that in the chart somehow